README.md

```diff
@@ -1,2 +1,97 @@
-# warp
-WDL Analysis Research Pipelines
+# Welcome to WARP
+WARP (WDL Analysis Research Pipelines) repository is a collection of cloud-optimized pipelines for processing biological data from the Broad Institute Data Sciences Platform and collaborators. 
+
+The contents of this repository are open source and released under the [BSD 3-Clause license](https://github.com/broadinstitute/warp/blob/master/LICENSE).
+
+### Table of Contents
+- [WARP Overview](#warp-overview)
+- [Navigating WARP](#navigating-warp)
+- [Using WARP](#using-warp)
+- [WARP Versioning and Releasing](#warp-versioning-and-releasing)
+- [Testing in WARP](#testing-in-warp)
+- [Feedback](#feedback)
+- [Acknowledgements](#acknowledgements)
+
+## WARP Overview
+WARP pipelines provide robust, standardized data analysis for the Broad Institute Genomics Platform and large consortia like the Human Cell Atlas and the BRAIN Initiative. You can count on WARP for rigorously scientifically validated, high scale, reproducible and open source pipelines. 
+
+Our pipelines are written as “workflows” using the [Workflow Description Language (WDL)](https://openwdl.org/) and they process a broad spectrum of “omic” and array-related datasets (see the overview table below).
+
+| Pipeline Category | Data Types |
+| --- | --- |
+| Germline Variant Discovery | Genomes, Exomes |
+| Genotyping Arrays | Variant discovery, Chip validation, Joint array analysis |
+| Single-cell/nuclei Transcriptomics | Droplet based (10x Genomics), Smartseq2 |
+| Single-cell Epigenomics | Single nuclei ATAC-seq, Single nuclei MethylC-seq |
+| Joint Genotyping | Genomes, Exomes | 
+| Somatic Alignment (beta) | Exomes |
+
+\
+*Ready to try the pipelines? You can get hands-on right away by testing our pipelines in [Terra](https://app.terra.bio/#workspaces?tagsFilter%5B0%5D=warp-pipelines&tab=public), a platform for collaborative cloud analysis! Learn how to get started in the [Using WARP](#2-run-the-pipeline-in-terra) section.*
+
+
```

Comment:
Maybe one too many new lines here?

Reply:
Updated to: "Try our pipelines in Terra, a platform for collaborative cloud analysis! Learn how in the Using WARP section."

Reply:
Ah - I like how it is now, but what I meant was the empty white space between this section and the next seemed like there was at least one more new line than necessary. But maybe that isn't rendered when the markdown is made?

Reply:
I put the extra space in because when I rendered the markdown, it felt like the message was getting squished below the table.